redwood-lazylayout-view/src/main/kotlin/app/cash/redwood/lazylayout/view/ViewLazyList.kt

```diff
@@ -99,7 +99,14 @@ internal open class ViewLazyList(context: Context) : LazyList<View> {
 
   final override val placeholder = Placeholders(recyclerView.recycledViewPool)
 
-  private val linearLayoutManager = LinearLayoutManager(recyclerView.context)
+  private val linearLayoutManager = object : LinearLayoutManager(recyclerView.context) {
+    // Identical to the implementation of [LinearLayout.generateDefaultLayoutParams].
```

Comment:
With a name like `LinearLayoutManager`, one would expect that at least the implementation of `generateDefaultLayoutParams()` would be the same.